selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/DashboardWorkspace.java

```diff
@@ -116,7 +116,7 @@ public String getName() {
     String RUN_WORKSPACE_BTN = "//button/span[text()='Run']";
     String STOP_WORKSPACE_BTN = "//button/span[contains(text(),'Stop')]";
     String DELETE_WORKSPACE_BTN = "//button/span[text()='Delete']";
-    String STATE_WORKSPACE = "//span[contains(@class, '%s')]";
+    String STATE_WORKSPACE = "//div[contains(@class, 'workspace-status')]/span[text()='%s']";
```

Comment:
It may be reasonable to use debug-id here instead of xpath,

Reply:
It will be done by https://github.com/eclipse/che/issues/6686.